game/engine.py

```
@@ -571,7 +585,15 @@ def get_address():
                 )
                 stdout, _ = proc.communicate(input=command, timeout=timeout_secs)
                 return stdout.strip()
-            except (subprocess.TimeoutExpired, OSError):
+            except subprocess.TimeoutExpired:
+                if proc:
+                    try:
+                        proc.kill()
+                        proc.communicate()
+                    except Exception:
+                        pass
```

Ruff (0.15.21)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)